### PR TITLE
Correct DocumentType publicId and systemId descriptions

### LINKED ISSUE #36448: Incomplete doctype property return value description under HTML
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/DocumentType

### What specific section or headline is this issue about?

Instance properties

### What information was incorrect, unhelpful, or incomplete?

The return values for the properties `doctype.publicId` and `.systemId` are stated to always be `""` in HTML.

### What did you expect to see?

This behavior as-written is only true if both are missing in the document type declaration string as written, i.e. if literally `<!DOCTYPE html>`.

However, if the public and/or system IDs *are* present (as, for example, on [http://www.w3.org/1999/xhtml](http://www.w3.org/1999/xhtml/)), then the returned strings will be set correspondingly.

### Do you have any supporting links, references, or citations?

HTML standard, [§13.2.6.4 The rules for parsing tokens in HTML content](https://html.spec.whatwg.org/multipage/parsing.html#parsing-main-inhtml)

> Append a [DocumentType](https://dom.spec.whatwg.org/#interface-documenttype) node to the [Document](https://html.spec.whatwg.org/multipage/dom.html#document) node, with its [name](https://dom.spec.whatwg.org/#concept-doctype-name) set to the name given in the DOCTYPE token, or the empty string if the name was missing; its [public ID](https://dom.spec.whatwg.org/#concept-doctype-publicid) set to the public identifier given in the DOCTYPE token, or the empty string if the public identifier was missing; and its [system ID](https://dom.spec.whatwg.org/#concept-doctype-systemid) set to the system identifier given in the DOCTYPE token, or the empty string if the system identifier was missing.

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/api/documenttype`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/DocumentType
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/documenttype/index.md
* Last commit: https://github.com/mdn/content/commit/bd15d43260b7e72b1066c04d9d9f3b79129c619c
* Document last modified: 2024-07-12T03:13:07.000Z

</details>

### PR DESCRIPTION
**Pull request by julia-aph**
### Description

Fix descriptions of the `DocumentType.publicId` and `DocumentType.systemId` instance properties to correct that they aren't always empty strings for HTML but correspond to the public and system IDs specified in the doctype string. Also fix the incorrect SVG system ID example.

### Related issues and pull requests

Fixes #36448

### CODE REVIEW

File: files/en-us/web/api/documenttype/index.md [hunk 1]
```diff
@@ -18,9 +18,9 @@ _Inherits properties from its parent, {{domxref("Node")}}._
 - {{domxref("DocumentType.name")}} {{ReadOnlyInline}}
   - : The type of the document. It is always `"html"` for HTML documents, but will vary for XML documents.
 - {{domxref("DocumentType.publicId")}} {{ReadOnlyInline}}
-  - : A string with an identifier of the type of document. Always empty (`""`) for HTML, it will be, for example, `"-//W3C//DTD SVG 1.1//EN"` for SVG documents.
+  - : A string with an identifier of the type of document. Empty (`""`) if the doctype string specifies no public ID, it will be, for example, `"-//W3C//DTD SVG 1.1//EN"` for SVG documents.
```
Comment (sideshowbarker) [inline files/en-us/web/api/documenttype/index.md:21]:
```suggestion
  - : A string with an identifier of the type of document. Empty if the doctype given specifies no public ID.
```

File: files/en-us/web/api/documenttype/index.md [hunk 2]
```diff
@@ -18,9 +18,9 @@ _Inherits properties from its parent, {{domxref("Node")}}._
...
 - {{domxref("DocumentType.systemId")}} {{ReadOnlyInline}}
-  - : A string containing the URL to the associated DTD. Always empty (`""`) for HTML, it will be, for example, `"http://www.w3.org/2000/svg"` for SVG documents.
+  - : A string containing the URL to the associated DTD. Empty (`""`) if the doctype string specifies no system ID, it will be, for example, `"http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"` for SVG documents.
```
Comment (sideshowbarker) [inline files/en-us/web/api/documenttype/index.md:23]:
```suggestion
  - : A string containing the URL to the associated DTD. Empty if the doctype given specifies no system ID.
```

**Review by sideshowbarker — APPROVED** [review]

**Comment by Josh-Cena** [comment]
Could you also fix the two property's pages? They look copied from the `name` page.

**Comment by sideshowbarker** [comment]
> Could you also fix the two property's pages? They look copied from the `name` page.

Yeah, it’d be good to have a follow-up PR to fix those

**Comment by sideshowbarker** [comment]
Julia, thanks much for catching this and fixing it, and congrats on landing your first docs change here — welcome aboard 🎉

**Pull request merged**